src/scss/core/_type.scss

```diff
@@ -0,0 +1,41 @@
+h1, .h1 {
+    font-size: $h1-font-size;
+    line-height: $h1-line-height;
+}
+
+h2, .h2 {
+    font-size: $h2-font-size;
+    line-height: $h2-line-height;
+}
+
+h3, .h3 {
+    font-size: $h3-font-size;
+    line-height: $h3-line-height;
+}
+
+h4, .h4 {
+    font-size: $h4-font-size;
+    line-height: $h4-line-height;
+}
```

Comment:
font-size values are automatically added by Bootstrap since we replaced its variables